grumphp.yml

```diff
@@ -1,13 +1,8 @@
-parameters:
-  git_hook_variables:
-    EXEC_GRUMPHP_COMMAND: docker-compose exec -T php
```

Comment:
Why are we removing the docker support by default?

Reply:
This was accidentally removed, my bad. Fixing it.